### ISSUE TITLE
Invalid token

### ISSUE THREAD

**Issue by peterorum** [piuccio/sublime-esformatter#50]
Hi,

line 22195 in esformatter.js has an invalid semi-colon at the end

```
var basepath = process.argv[2].replace(/\\/g, "/");,
    options = JSON.parse(process.argv[3] || "{}");
```

I also found I now had to have an .esformatter file in my path or there would be an error on line 174.
```
      throw new TypeError('Arguments to path.join must be strings');
```

**Comment by piuccio** [comment]
:facepalm: I swear I tested this.
Or not :tongue:

**Comment by piuccio** [comment]
I believe this is fixed in `1.6.4`
